### PR TITLE
config: Use `main` branch instead of `master`

### PR DESCRIPTION
**Pull request by fidencio**
It seems that the only required change is to switch the backend branch
from `master` to `main`, and theoretically we should be good to go to
fully switch to using `main` on *all* repos which are part of the
kata-containers organization.

Signed-off-by: Fabiano Fidêncio <fidencio@redhat.com>

### CODE REVIEW

**Review by jodh-intel — APPROVED** [review]
Thanks @fidencio. lgtm but I don't know anything about how the site is generated so @jimmytipit  will need to approve before we land this.

**Comment by fidencio** [comment]
> Thanks @fidencio. lgtm but I don't know anything about how the site is generated so @jimmytipit will need to approve before we land this.

I've contacted @jimmytipit both here and on Slack.  Is there an easier / more reliabe way to reach Jimmy?

**Comment by jodh-intel** [comment]
He may be on holiday. If so, maybe @ttx knows about this?

**Comment by ttx** [comment]
Sorry for the delay, I'll make sure he sees this.

**Comment by jodh-intel** [comment]
Thanks very much @ttx :smile:

**Comment by jimmytipit** [comment]
Hi all - I apologize for missing this.  Approving now.

**Comment by jodh-intel** [comment]
Thanks @jimmytipit.

**Pull request merged**
